spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/BatchDataReader.java

```diff
@@ -49,6 +53,7 @@ protected Stream<ContentFile<?>> referencedFiles(FileScanTask task) {
   @Override
   protected CloseableIterator<ColumnarBatch> open(FileScanTask task) {
     String filePath = task.file().path().toString();
+    LOG.debug("Opening data file {}", filePath);
```

Comment:
Nit: This debug log could arguably be removed, but I wouldn’t block on this comment. It could be good to have more verbose debug logs.